From 0ccbaa134093bf6afc79f2d20d061bca5a8754ed Mon Sep 17 00:00:00 2001
From: Kai Kang <kai.kang@windriver.com>
Date: Thu, 29 Mar 2018 16:02:05 +0800
Subject: [PATCH 04/10] avoid host contamination

Remove hardcode path refer to host to avoid host contamination.

Upstream-Status: Pending
Signed-off-by: Kai Kang <kai.kang@windriver.com>

Rebase to 9.23
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 devices/devs.mak | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/devices/devs.mak b/devices/devs.mak
index 846aa50..9570182 100644
--- a/devices/devs.mak
+++ b/devices/devs.mak
@@ -393,7 +393,7 @@ $(DEVOBJ)gdevxalt.$(OBJ) : $(DEVSRC)gdevxalt.c $(GDEVX) $(math__h) $(memory__h)\
 ### NON PORTABLE, ONLY UNIX WITH GCC SUPPORT
 
 $(DEVOBJ)X11.so : $(x11alt_) $(x11_) $(DEVS_MAK) $(MAKEDIRS)
-	$(CCLD) $(LDFLAGS) -shared -o $(DEVOBJ)X11.so $(x11alt_) $(x11_) -L/usr/X11R6/lib -lXt -lSM -lICE -lXext -lX11 $(XLIBDIRS)
+	$(CCLD) $(LDFLAGS) -shared -o $(DEVOBJ)X11.so $(x11alt_) $(x11_) -lXt -lSM -lICE -lXext -lX11 $(XLIBDIRS)
 
 ###### --------------- Memory-buffered printer devices --------------- ######
 
-- 
1.8.3.1

